src/ontology/edits/oeo-model.omn

```diff
@@ -4283,3 +4402,97 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/824",
         OEO_00140167
     
     
+Individual: OEO_00400014
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1829
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Ambigous uncertainty is a nature of uncertainty that stems from multiple frames of reference about a phenomenon existing similtaneously.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "ambiguity of uncertainty"@en,
```

Comment:
<http://purl.obolibrary.org/obo/IAO_0000118> "ambiguity of uncertainty"@en,
        <http://purl.obolibrary.org/obo/IAO_0000119> "https://www.researchgate.net/publication/264816512_Classifying_and_communicating_uncertainties_in_model-based_policy_analysis",
        rdfs:label "ambiguous uncertainty"@en